src/helpers/menuNavBar.ts

```
@@ -7,33 +7,33 @@ type Menu = {
   onClick?: () => void;
 };
 
-const subMenuListasNominais: Menu[] = [
+const subMenuListasNominais = (visao : string) : Menu[]=> [
     {
         label: "Citopatológico",
-        url: "/busca-ativa/citopatologico",
+        url: `/busca-ativa/citopatologico?aba=${''}&sub_aba=0&visao=${visao}`,
         onClick: () => mixpanel.track("menu_click", {"menu_action": "acessar_lista_citopatologico"})
```

Biome (1.9.4)

[error] This object property name part should be in camelCase.



(lint/style/useNamingConvention)